### PR TITLE
Liif config

### PR DESCRIPTION
**Pull request by Yshuo-Li**
Add config file of LIIF.

### CODE REVIEW

**Comment by innerlee** [comment]
Need rebase now

File: configs/restorers/liif/liif_edsr_norm_c64b16_g1_1000k_div2k.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+# dataset settings
+train_dataset_type = 'SRFolderGTDataset'
+val_dataset_type = 'SRFolderGTDataset'
+test_dataset_type = 'SRFolderGTDataset'
+train_pipeline = [
+    dict(
+        type='LoadImageFromFile',
+        io_backend='disk',
+        key='gt',
+        flag='unchanged',
```
Comment (innerlee) [inline configs/restorers/liif/liif_edsr_norm_c64b16_g1_1000k_div2k.py:40]: What the reason to use `unchanged`?

File: configs/restorers/liif/liif_edsr_norm_x2~4_c64b16_g1_1000k_div2k.py [hunk 2]
```diff
@@ -0,0 +1,168 @@
+exp_name = 'liif_edsr_norm_x2~4_c64b16_g1_1000k_div2k'
```
Comment (innerlee) [inline configs/restorers/liif/liif_edsr_norm_x2~4_c64b16_g1_1000k_div2k.py:1]: avoid `~` in filename

Comment (innerlee) [reply configs/restorers/liif/liif_edsr_norm_x2~4_c64b16_g1_1000k_div2k.py:1]: `-` is ok

File: configs/restorers/liif/liif_edsr_norm_x2~4_c64b16_g1_1000k_div2k.py [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        num_blocks=16),
+    imnet=dict(
+        type='MLPRefiner',
+        in_dim=64,
+        out_dim=3,
+        hidden_list=[256, 256, 256, 256]),
+    local_ensemble=True,
+    feat_unfold=True,
+    cell_decode=True,
+    data_mean=[0.4488, 0.4371, 0.4040],
```
Comment (innerlee) [inline configs/restorers/liif/liif_edsr_norm_x2~4_c64b16_g1_1000k_div2k.py:21]: rgb_mean

File: configs/restorers/liif/liif_edsr_norm_x2~4_c64b16_g1_1000k_div2k.py [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+    dict(
+        type='LoadImageFromFile',
+        io_backend='disk',
+        key='gt',
+        flag='unchanged',
+        channel_order='rgb'),
+    dict(type='RandomDownSampling', scale_min=scale_max, scale_max=scale_max),
+    dict(type='RescaleToZeroOne', keys=['lq', 'gt']),
+    dict(
+        type='Normalize',
```
Comment (innerlee) [inline configs/restorers/liif/liif_edsr_norm_x2~4_c64b16_g1_1000k_div2k.py:76]: Can this be removed?

File: configs/restorers/liif/liif_edsr_norm_c64b16_g1_1000k_div2k.py (hunk 1 above)
Comment (Yshuo-Li) [reply configs/restorers/liif/liif_edsr_norm_c64b16_g1_1000k_div2k.py:40]: `color` is ok, too. Will be modified to `color`.

Comment (innerlee) [reply configs/restorers/liif/liif_edsr_norm_c64b16_g1_1000k_div2k.py:40]: They are different, so its better to be clear and exact

**Review by innerlee — APPROVED** [review]

**Pull request merged**